includes/payment-methods/class-wc-stripe-upe-payment-method-multibanco.php

```diff
@@ -29,6 +29,104 @@ public function __construct() {
 		);
 
 		add_filter( 'wc_stripe_allowed_payment_processing_statuses', [ $this, 'add_allowed_payment_processing_statuses' ], 10, 2 );
+
+		add_action( 'wc_gateway_stripe_process_payment_intent_requires_action', [ $this, 'save_instructions' ], 10, 2 );
+		add_action( 'woocommerce_thankyou_stripe_multibanco', [ $this, 'thankyou_page' ] );
+		add_action( 'woocommerce_email_before_order_table', [ $this, 'email_instructions' ], 10, 3 );
+	}
+
+	/**
+	 * Output for the order received page.
+	 *
+	 * @param int $order_id
+	 */
+	public function thankyou_page( $order_id ) {
+		$order = wc_get_order( $order_id );
+		if ( ! $order ) {
+			return;
+		}
+
+		$this->get_instructions( $order );
+	}
+
+	/**
+	 * Add content to the WC emails.
+	 *
+	 * @param WC_Order $order
+	 * @param bool     $sent_to_admin
+	 * @param bool     $plain_text
+	 */
+	public function email_instructions( $order, $sent_to_admin, $plain_text = false ) {
+		$payment_method = $order->get_payment_method();
+		if ( ! $sent_to_admin && 'stripe_multibanco' === $payment_method && $order->has_status( 'on-hold' ) ) {
+			$this->get_instructions( $order, $plain_text );
+		}
+	}
+
+	/**
+	 * Gets Multibanco payment instructions for the customer.
+	 *
+	 * @param WC_Order $order
+	 */
+	public function get_instructions( $order, $plain_text = false ) {
```

Comment:
In [legacy experience](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/3c8c8de87421ba65808bcf8583137ceb4e674127/includes/payment-methods/class-wc-gateway-stripe-multibanco.php#L243), the instructions were in Portuguese. But here, they are in English. Is this intentional?

Reply:
Yes, I decided to use English text in the code since we are using `esc_html_e()` to translate to the determined locale anyways, and Stripe docs say Multibanco is available in Europe and US.